packages/perseus/src/widgets/label-image/label-image.tsx

```diff
@@ -67,23 +68,14 @@ type Point = {
     y: number;
 };
 
-// TODO: should this be using WidgetProps / PerseusLabelImageWidgetOptions?
 type LabelImageProps = ChangeableProps &
-    DependencyProps & {
+    DependencyProps &
+    // TODO: there's some weirdness in our types between
+    // PerseusLabelImageMarker and InteractiveMarkerType
+    Omit<PerseusLabelImageWidgetOptions, "markers"> & {
```

Comment:
It looks like `MarkerType` in `label-image/types.ts` duplicates `PerseusLabelImageMarker` in `perseus-types.ts`. `InteractiveMarkerType` just adds some optional properties to `MarkerType`. Those optional properties seem like they're probably set when the widget emits state changes (through `onChange`).